week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,6 +1,8 @@
 // Declare your function here
-
-const greeting = createLongGreeting("Daniel", 30);
+function createLongGreeting(a, b) {
+  return "Hello,my name is " + a + " and I am " + b + " years old.";
+}
```

Comment:
Try to use more meaningful names for the parameters unless they really are abstract values. Something like `name` and `age` would make more sense here than `a` and `b`.